root/etc/s6-overlay/s6-rc.d/init-mod-swag-crowdsec/run

```diff
@@ -28,29 +22,11 @@ echo "\
     lua-sec \
     nginx-mod-http-lua" >> /mod-repo-packages-to-install.list
 
-# Download nginx bouncer
-if [[ -z ${CROWDSEC_VERSION+x} ]]; then \
-    CROWDSEC_VERSION=$(curl -s "https://api.github.com/repos/crowdsecurity/cs-nginx-bouncer/releases/latest" | awk '/tag_name/{print $4;exit}' FS='[""]');
-fi
-
-if [[ -z ${CROWDSEC_VERSION+x} ]]; then \
-    echo "**** Could not fetch current bouncer version from Github ****"
-    exit 1
-fi
-
-curl "${CURL_NOISE_LEVEL}" -Lo \
-    /tmp/crowdsec.tar.gz -L \
-    "https://github.com/crowdsecurity/cs-nginx-bouncer/releases/download/${CROWDSEC_VERSION}/crowdsec-nginx-bouncer.tgz"
-
+# Extract nginx bouncer
 mkdir -p /tmp/crowdsec
 
-if ! tar -tzf /tmp/crowdsec.tar.gz >/dev/null 2>&1; then
-    echo "**** Invalid tarball, could not download crowdsec bouncer ****"
-    exit 1
-fi
-
 tar xf \
-    /tmp/crowdsec.tar.gz -C \
+    /crowdsec-nginx-bouncer.tgz -C \
```

Comment:
I think this would fail if the container is restarted without a mod update as the tarball is deleted at the end of the script.

Reply:
Fixed.